eth2near/eth2near-block-relay-rs/src/beacon_rpc_client.rs

```diff
@@ -469,16 +469,21 @@ impl BeaconRPCClient {
 #[cfg(test)]
 mod tests {
     use crate::beacon_rpc_client::BeaconRPCClient;
+    use crate::config_for_tests::ConfigForTests;
     use crate::test_utils::read_json_file_from_data_dir;
+    use contract_wrapper::utils::trim_quotes;
+    use serde_json::Value;
     use types::BeaconBlockBody;
     use types::BeaconBlockHeader;
     use types::MainnetEthSpec;
 
-    const TEST_BEACON_BLOCK_ID: u32 = 741888;
-    const BEACON_ENDPOINT: &str = "https://lodestar-kiln.chainsafe.io";
     const TIMEOUT_SECONDS: u64 = 30;
     const TIMEOUT_STATE_SECONDS: u64 = 1000;
 
+    fn get_config() -> ConfigForTests {
+        ConfigForTests::load_from_toml("config_for_tests.toml".try_into().unwrap())
+    }
```

Comment:
Please rename to `get_test_config()`

Reply:
fixed https://github.com/aurora-is-near/rainbow-bridge/pull/818/commits/dc111e862fb4e2c1e377271839933d7642ac1fee